extension/opampextension/go.mod

```diff
@@ -111,7 +111,7 @@ require (
 	github.com/tklauser/go-sysconf v0.3.14 // indirect
 	github.com/tklauser/numcpus v0.8.0 // indirect
 	github.com/yusufpapurcu/wmi v1.2.4 // indirect
-	go.opentelemetry.io/collector/pdata v1.28.2-0.20250319144947-41a9ea7f7402 // indirect
+	go.opentelemetry.io/collector/pdata v1.28.2-0.20250319144947-41a9ea7f7402
```

Comment:
this change seems weird

Reply:
yeah this was weird, this was after a `go mod tidy`, lemme try again.

Reply:
oh this is because im storing the pcommon.Map

Reply:
i can revert that and just convert it to `map[string]string` as needed, thoughts?

Reply:
@codeboten there was no good way around this if i wanted to avoid doing a switch case on the type for the any returned by the pcommon `AsRaw`. Let me know if you'd prefer that approach